### DISCUSSION TITLE
hacking-in-disassembly

### DISCUSSION THREAD

**Discussion by giscus** [SethBarberee/sethbarberee.github.io#17] [Announcements]
# hacking-in-disassembly

It’s been a fun time this summer. I’ve been messing with the disassemblies of Pokemon FireRed and Emerald. I have some basic C knowledge and Assembly knowledge so I was able to figure out, for the most part, how the code worked. FireRed was harder to change the starters vs Emerald due to having to edit Text, Pokemon, and Trainer Parties. I only edited the first battle and didn’t update the rest of the game. If I was doing an actual hack, I probably would have done that. This mainly took an hour once I figured out what I was doing.

https://sethbarberee.github.io/hacking-in-disassembly

**Comment by theSoberSobber** [comment]
guess you forgot?

**Reply by SethBarberee** [reply]
Haha yeah. Will get that done when I’m back in the states from my vacation. Have a growing todo list of other things I need to do on the blog.

**Reply by theSoberSobber** [reply]
cool
